projects/packages/jitm/src/class-jitm.php

```diff
@@ -255,21 +255,47 @@ public function ajax_message() {
 	 * @param bool   $full_jp_logo_exists Is there a big JP logo already displayed on this screen.
 	 */
 	public function generate_icon( $content_icon, $full_jp_logo_exists ) {
+		$date_now   = new \DateTime();
+		$feb_4_date = new \DateTime( '02-04-2025' );
```

Comment:
Would it be worth adding `wp_timezone()` when getting the dates, so as to switch at the right moment regardless of your site settings?

Reply:
Thanks for the suggestion @jeherve! Yes sure, why not! It wouldn't hurt!
I mean AFAIK I don't think it's completely necessary for the logo to switch for everybody at the exact same time, but then again, sure why not? I think it would be even better that way! 😉

But would you mind checking/reviewing my code changes using `wp_timezone()`?
I don't think I've ever used `wp_timezone()` before, so I'm not completely sure I did it correctly..., Does it look OK to you?

Reply:
> AFAIK I don't think it's completely necessary for the logo to switch for everybody at the exact same time

Can confirm. As long as it happens at some point on the 4th :)